### PR TITLE
Allow comments in module JSON

### PR DESCRIPTION
**Pull request by V1shvesh**

### CODE REVIEW

**Comment by serin-delaunay** [comment]
That's great, but it only works for the `CornerCombiner` module. Could you also enable comments for the rest of the modules? If you put the `"comment"` schema inside `#/definitions`, then this would be as simple as adding `"comment": { "$ref": "#/definitions/comment"}` to each of the module schemas.

**Comment by V1shvesh** [comment]
Sure. Thanks for the help!

**Comment by V1shvesh** [comment]
Comment for `"cache"' required??

**Comment by serin-delaunay** [comment]
Yes.

**Comment by V1shvesh** [comment]
Check it now perhaps...

**Comment by serin-delaunay** [comment]
Thanks! I see why you asked about those four modules - they haven't been added to the `#/modules` schema. If you fix that bug, this will be ready to merge.

**Comment by V1shvesh** [comment]
Done! :+1:

File: doc/schemas/module_group_schema.json [hunk 1]
```diff
@@ -52,6 +55,7 @@
           {"$ref": "#/definitions/pow"},
           {"$ref": "#/definitions/quadrant_selector"},
           {"$ref": "#/definitions/ridged_multi"},
+          {"$ref": "#/definitions/reciprocal"},
```
Comment (serin-delaunay) [inline doc/schemas/module_group_schema.json]: `reciprocal` should come before `ridged_multi`.

Comment (V1shvesh) [reply doc/schemas/module_group_schema.json]: Didn't know how I missed that...

**Comment by V1shvesh** [comment]
Reordered `reciprocal`

**Comment by serin-delaunay** [comment]
Thanks, great work. Merged.

**Comment by V1shvesh** [comment]
Thanks to you as well! This was absolutely amazing!

**Pull request merged**